examples/pane_grid/src/main.rs

```diff
@@ -82,37 +102,54 @@ impl Sandbox for Example {
             }
             Message::Dragged(_) => {}
             Message::Close(pane) => {
-                let _ = self.panes.close(&pane);
+                if let Some((_, sibling)) = self.panes.close(&pane) {
+                    self.focus = Some(sibling);
+                }
             }
             Message::CloseFocused => {
-                if let Some(pane) = self.panes.active() {
-                    let _ = self.panes.close(&pane);
+                if let Some(pane) = self.focus {
+                    if let Some((_, sibling)) = self.panes.close(&pane) {
+                        self.focus = Some(sibling);
+                    }
                 }
             }
         }
+
+        Command::none()
+    }
+
+    fn subscription(&self) -> Subscription<Message> {
+        subscription::events_with(|event| match event {
+            Event::Keyboard(keyboard::Event::KeyPressed {
+                modifiers,
+                key_code,
+            }) if modifiers.control => handle_hotkey(key_code),
+            _ => None,
```

Comment:
May I propose a platform-specific modifier key?

```rust
Event::Keyboard(keyboard::Event::KeyPressed {
    modifiers,
    key_code,
}) => {
    #[cfg(target_os = "macos")]
    let modifier = modifiers.logo;

    #[cfg(not(target_os = "macos"))]
    let modifier = modifiers.control;

    if modifier {
        handle_hotkey(key_code)
    } else {
        None
    }
}
_ => None,
```

Reply:
Done in 86fa12229e7117306b8297a09aa22c99802600c8. I have introduced an additional method to `ModifiersState` to help us avoid hardcoding this kind of logic.

Reply:
Nice! Love taking care of that at the source rather than at the edge 👌